framework/contracts/account/src/error.rs

```diff
@@ -8,25 +8,22 @@ use thiserror::Error;
 
 #[derive(Error, Debug, PartialEq)]
 pub enum AccountError {
-    #[error("{0}")]
+    #[error(transparent)]
     Std(#[from] StdError),
 
-    #[error("{0}")]
+    #[error(transparent)]
     Abstract(#[from] AbstractError),
 
-    #[error("{0}")]
-    AbstractSdk(#[from] AbstractSdkError),
-
-    #[error("{0}")]
+    #[error(transparent)]
     Validation(#[from] ValidationError),
 
-    #[error("{0}")]
+    #[error(transparent)]
     Ownership(#[from] abstract_std::objects::ownership::GovOwnershipError),
 
-    #[error("{0}")]
+    #[error(transparent)]
     Instantiate2AddressError(#[from] Instantiate2AddressError),
 
-    #[error("{0}")]
+    #[error(transparent)]
     RegistryError(#[from] RegistryError),
```

Comment:
Not sure why it's replaced from "{0}" to transparent in this file, but not the others

Reply:
Because I did only the first file and forgot the other ones.